src/System.Text.Encoding/tests/Decoder/Decoder.netcoreapp.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
nit: looks this file testing span APIs only. we may have the file name include Span word (e.g. DecoderSpanTests.netcoreapp.cs)